src/it/scala/com/blackmorse/spark/clickhouse/cluster/ClusterShardedReaderTest.scala

```diff
@@ -22,6 +22,108 @@ class ClusterShardedReaderTest extends AnyFlatSpec with Matchers with DataFrameS
     shardData1 ++ shardData2
   }
 
+  private def writeDuplicateData: Seq[(Int, String)] = {
```

Comment:
Probably we should proceed with one generic method for writing the data? 

Signature like this:

```
def writeData[T](shard1Data: Seq[T], shard2Data: Seq[T])(implicit e: Encoder[T]): Seq[T]
```

Reply:
Done